src/runner/index.js

```diff
@@ -547,6 +548,31 @@ export default class Runner extends EventEmitter {
         return this._setBootstrapperOptions();
     }
 
+    async _initReporters (reporters) {
+        return Promise.all(reporters.map(reporter => reporter.init()))
+            .then(initResults => {
+                const reporterInitErrors = [];
+
+                initResults.forEach(({ reporter, error }) => {
+                    if (error) {
+                        reporterInitErrors.push(`- "${reporter.plugin.name}": ${error.message}`);
+
+                        remove(reporters, reporter);
+                    }
+                });
+
+                const cannotInitAllReporters = initResults.length === reporterInitErrors.length;
+                const reporterInitErrMessage = reporterInitErrors.join('\n');
+
+                if (reporterInitErrors.length) {
+                    if (cannotInitAllReporters)
```

Comment:
I don't feel that is this the correct way of handling reporter errors. The most common case of using multiple reporters now is using the spec reporter for the CI build log and something like the xunit reporter to make a good looking report in the CI interface. If the xunit reporter fails to initialize, the build will be still failed since CI won't be able to find the report file in the end, and a lot of CI time would be wasted.

Moreover, it contradicts with the our error handling policy in other reporters methods: if an error happens in any other method, we always stop tests regardless if we have a single or multiple reporters.